tests/src/KeyManagerTest.php

```diff
@@ -103,14 +103,31 @@ public function testGetDefaultKey() {
    * Test load of defaul key content.
    *
    * @group key
-   *
-   * @todo There is no Key::getContents() method so this will return an error.
    */
   public function testGetDefaultKeyContent() {
+    // On the first run, config storage will return NULL.
```

Comment:
Much of this is borrowed from `testGetDefaultKey`. Would it be better to consolidate the test data into a provider method?